src/dream.mli

```diff
@@ -414,7 +414,7 @@ https://github.com/aantron/dream/issues
 (** Replaces the path. See {!Dream.val-path}. *)
 (**/**)
 
-val query : request -> string -> string option
+val query : request -> string -> ?filter_query:bool -> unit -> string option
```

Comment:
If you move the optional parameter to the left you can skip the final unit argument. Then you don't have to change the examples. 🙂

Reply:
Beautiful, thanks! 🤩